### PR TITLE
Move process event to FizzServerHandshake

### PR DESCRIPTION
**Pull request by deadalnix**
Move the management of pending crypto event - inherently fizz specific - to FizzServerHandshake.

Depends on #175

### CODE REVIEW

File: quic/fizz/server/handshake/FizzServerHandshake.h [hunk 1]
```diff
@@ -37,10 +37,16 @@ class FizzServerHandshake : public ServerHandshake {
       std::unique_ptr<fizz::server::AppTokenValidator> validator) override;
 
   EncryptionLevel getReadRecordLayerEncryptionLevel() override;
+  void processSocketData(folly::IOBufQueue& queue) override;
   std::pair<std::unique_ptr<Aead>, std::unique_ptr<PacketNumberCipher>>
   buildCiphers(folly::ByteRange secret) override;
 
-  void processSocketData(folly::IOBufQueue& queue) override;
+  void processAccept() override;
+  bool processPendingCryptoEvent() override;
```
Comment (yangchi) [inline quic/fizz/server/handshake/FizzServerHandshake.h:45]: document what this returns

**Comment by yangchi** [comment]
I accepted this on Phabricator. No need to hold this much longer just for one nit comment.

**Comment by deadalnix** [comment]
Do you want me to submit a second patch adding a comment?

**Comment by yangchi** [comment]
@deadalnix Up to you i guess, it's a tiny nit. If you want to add the comment, do it in a separate PR so that @mjoras doesn't have to import this one again.

**Comment by deadalnix** [comment]
ok. I still have more work to do on that one as there are still fizz element in the ServerHandshake, so let's see where the whole design end up. This function might be gone altogether by the end of it.

**Pull request closed without merging**